### PR TITLE
Updated @grpc/grpc-js, readable-stream

### PR DESCRIPTION
**Pull request by mrickard**
# Proposed Release Notes
Upgraded @grpc/grpc-js from 1.8.9 to 1.8.10
Upgraded readable-stream from 3.6.0 to 3.6.1

## Links
https://issues.newrelic.com/browse/NEWRELIC-7616

## Details
Snyk PRs were failing versioned tests. This PR is an attempt to re-run and consolidate changes.

Closes NEWRELIC-7616

### CODE REVIEW

File: THIRD_PARTY_NOTICES.md [hunk 1]
```diff
@@ -74,6 +74,7 @@ code, the source code can be found at [https://github.com/newrelic/node-newrelic
 **[optionalDependencies](#optionalDependencies)**
 
 * [@contrast/fn-inspect](#contrastfn-inspect)
+* [@newrelic/native-metrics](#newrelicnative-metrics)
```
Comment (jordigh) [inline THIRD_PARTY_NOTICES.md:77]: Something is funny here, this isn't a third-party package.

**Review by jordigh — CHANGES_REQUESTED** [review]

File: THIRD_PARTY_NOTICES.md (hunk 1 above)
Comment (mrickard) [reply THIRD_PARTY_NOTICES.md:77]: It's weird--I didn't make or commit those changes, but they're in my local git history.

Comment (mrickard) [reply THIRD_PARTY_NOTICES.md:77]: I'm going to try this again...

Comment (mrickard) [reply THIRD_PARTY_NOTICES.md:77]: Committing package.json and package-lock.json is triggering the husky pre-commit hook. That's what's generating those changes.

**Review by jordigh — APPROVED** [review]

**Pull request merged**